lib/Doctrine/Common/Util/ClassUtils.php

```diff
@@ -9,6 +9,8 @@
  *
  * @author Benjamin Eberlei <kontakt@beberlei.de>
  * @author Johannes Schmitt <schmittjoh@gmail.com>
+ *
+ * @deprecated The ClassUtils class is deprecated.
```

Comment:
The `ClassUtils` is useful to me. What would you suggest in replacement?

Reply:
Note: Was useful to resolve proxy classes and get the real class name.

Reply:
This will no longer be applicable, because class resolution goes to the `ProxyManager` component for ORM 3.x

Reply:
@Ocramius So if I well understood, we have to keep the usage of this deprecated `ClassUtils` unti ORM v3 usage and then go to the ProxyManager class ?

Reply:
Correct. In general, working with `get_class()` is discourage, but if you need it, you will probably have to use both classes during a transition period.